www/src/templates/template-blog-post.js

```diff
@@ -146,7 +146,16 @@ class BlogPostTemplate extends React.Component {
                   flex: `0 0 auto`,
                 }}
               >
-                <Link to={post.frontmatter.author.fields.slug}>
+                <Link
+                  to={post.frontmatter.author.fields.slug}
+                  css={{
+                    borderBottom: `0 !important`,
```

Comment:
Do all of these _need_ `!important`? I wouldn't have thought so, they shouldn't need extra specificity here.

Reply:
Without the `!important` it's overridden by other CSS. Not sure how we can set the emotion-js inline CSS as default.

Reply:
It's here -> https://github.com/gatsbyjs/gatsby/blob/master/www/src/utils/typography.js#L169

So (annoyingly) what's happening here is that .main-body a is _more specific_ than what emotion is doing, which will be like `.asdf-1234 { borderBottom: 0; }`

Can we solve this problem without using `!important`? One way you _should_ be able to do it is hoist the css up a level, and just use something like

```js
css={{
  a: {
    borderBottom: 0,
    boxShadow: 'none',
    ':hover': {
      background: 'none'
    }
  }
}}
```

Reply:
I guess `!important` is fine too. It just sorta negates some of the benefit of using CSS in JS in the first place, hah.

Reply:
Another solution might be using the size of the avatar image for the link. I updated the code, no `!important` needed anymore. 
If this is ok? 😄

Reply:
Most of the times something like https://github.com/gatsbyjs/gatsby/blob/fc81ecaa4e4b2b0b0b94dfb9337aad5866569726/www/src/components/package-readme.js#L34-L49 should work fine, e.g. 

```javascript
css={{
  "&&": {
  ...styles where specificity needs to be increased
}}
´´´

See https://github.com/emotion-js/emotion/issues/724#issue-332088759 for another example + generated `className`.

Reply:
Nice trick @fk!! @gurtjun would you want to try that?

Reply:
Good catch @fk! That seems to work. Used the ampersands in f66f114